src/WooCommerceCheckout.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * WooCommerce Checkout related functionality.
+ */
+class WooCommerceCheckout {
+
+  /**
+   * WooCommerce Checkout initialization method.
+   */
+  public static function init() {
+    add_filter('woocommerce_get_settings_shop_standards', __CLASS__ . '::woocommerceCheckoutSettings');
+
+    if (is_admin()) {
+      return;
+    }
+
+    // Adds confirmation email field to the checkout page.
+    if (get_option('_' . Plugin::L10N . '_checkout_email_confirmation_field') === 'yes') {
+      add_filter('woocommerce_checkout_fields', __CLASS__ . '::addConfirmationEmailCheckoutField');
+      add_action('woocommerce_checkout_process', __CLASS__ . '::checkConfirmationEmailField');
+    }
+  }
+
+  /**
+   * Adds WooCommerce Checkout specific backend settings.
+   *
+   * @implements woocommerce_get_settings_shop_standards
+   */
+  public static function woocommerceCheckoutSettings(array $settings): array {
+    $settings[] = [
+      'type' => 'title',
+      'name' => __('Checkout settings', Plugin::L10N),
+    ];
+    $settings[] = [
+      'type' => 'checkbox',
+      'id' => '_' . Plugin::L10N . '_checkout_email_confirmation_field',
+      'name' => __('Enable e-mail confirmation field', Plugin::L10N),
```

Comment:
A confirmation is something else – usually means an actual verification email containing a link to click.
```suggestion
      'name' => __('Require users to enter billing email address twice', Plugin::L10N),
```

Reply:
@stephansperling — should I change it?

Reply:
Please! Fine for me!